### DISCUSSION TITLE
WILL NOT WORK WHEN TRYING TO IMPORT .utm FILE.

### DISCUSSION THREAD

**Discussion by Sahelalemi** [utmapp/UTM#5840] [UTM Help (iOS)]
ok so basically when I click on a .utm file to import it where it says import prebuilt, nothing happens. I don’t know why but it’s super annoying.

**Configuration (required)**
* UTM Version: v4.4.4
* OS Version: 17.0.3/1
* Device Model: iPad 7th gen
* Is it jailbroken (name jailbreak used)? No
* How did you install UTM? Appdb

**Crash log**
If the app crashed, you need a crash log. To get your crash log, open the Settings app and browse to `Privacy -> Analytics & Improvements -> Analytics Data` and find the latest entry for UTM. You should export the text and attach it here.

**Debug log**
For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the bottom of the configuration options (or at the top of the `QEMU` page) is `Debug Log`. Turn it on and save the VM. After you experience the issue, quit the VM and re-launch UTM. Open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since GitHub has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by osy** [comment]
These are the only supported way of installing UTM. Most other ways of signing is broken. https://docs.getutm.app/installation/ios/

**Reply by Sahelalemi** [reply]
Why tho.. I can’t afford a pc..

**Reply by osy** [reply]
Like I said, these services do not sign the IPA correctly. There is nothing we can do. Contact them if you're having issues.

**Reply by Sahelalemi** [reply]
So apart from altstore could there be a signing certificate thing for free?

**Reply by osy** [reply]
I don't know, ask Reddit or something.

**Reply by Sahelalemi** [reply]
Ok do you know where I can get a p12 cert? For free

**Reply by Sahelalemi** [reply]
And mobileprovision

**Reply by Sahelalemi** [reply]
Oh I fixed it basically you out your .utm filed in the “UTM” folder in files.
